### ISSUE TITLE
Provide a more descriptive error upon executing as sudo/root when bower fails.

### ISSUE THREAD

**Issue by bmckean** [linnovate/mean-cli#86]
A simple error message that explains that it must be run without sudo/root will be less confusing for new users.

**Comment by liorkesos** [comment]
I agree. I think we should not let users run mean as root (maybe bake in a -f flag to overide) but the defaults should bail out.
Tons of support for mixed permissions related problems in npm etc..
We might also try to seek his situation out and tell the user about it.
